components/movementsensor/gpsrtkpmtk/gpsrtkpmtk.go

```diff
@@ -682,3 +698,80 @@ func (g *rtkI2C) Close(ctx context.Context) error {
 
 	return nil
 }
+
+// sendGGAMessage sends GGA messages to the serial port. This is only used to get NTRIP stream
+// from a virtual reference point.
+func (g *rtkI2C) sendGGAMessage() {
+	// Open an I2C handle to read NMEA messages
+	handle, err := g.bus.OpenHandle(g.addr)
+	if err != nil {
+		g.logger.Errorf("can't open gps i2c handle: %s", err)
+		g.err.Set(err)
+		return
+	}
+	defer func() {
+		if closeErr := handle.Close(); closeErr != nil {
+			g.logger.Errorf("error while closing I2C handle after reading GGA message: %v", closeErr)
+			g.err.Set(closeErr)
+		}
+	}()
+
+	// Create a buffer to collect NMEA messages
+	messageBuffer := make([]byte, 0, 1024)
+
+	for !g.ntripStatus {
+		select {
+		case <-g.cancelCtx.Done():
+			return
+		default:
+		}
+
+		// Read from the I2C device
+		buffer, err := handle.Read(g.cancelCtx, 1024)
+		if err != nil {
+			g.err.Set(err)
+			return
+		}
+
+		// Process the received data
+		for _, b := range buffer {
+			// PMTK uses CRLF line endings to terminate sentences, but just LF to blank data.
```

Comment:
> just LF to blank data.

Did you mean "just LF _for_ blank data"? or "just LF to _clear_ data"? I can't tell what this means. The next line makes more sense: I wonder if it should be

> PMTK uses CRLF line endings at the end of sentences, but LF to distinguish data in the middle. We ignore line feeds and look for the carriage return to mark the end of the sentence.